internal/pkg/copy/copy.go

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
Ignore this one too - from https://gist.github.com/m4ng0squ4sh/92462b38df26839a3ca324697c8cba04